src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```
@@ -1581,6 +1582,11 @@ services:
 
     form.type.sell.product.data_transformer.typeahead_redirection_target_transformer:
         class: 'PrestaShopBundle\Form\Admin\Sell\Product\DataTransformer\TypeaheadRedirectionTargetTransformer'
+        deprecated: 'The "%service_id%" service is deprecated since 1.7.9 and will be removed in next major.'
```

```suggestion
        deprecated: 'The "%service_id%" service is deprecated since 1.7.9.0 and will be removed in next major.'
```
Until we change to semver

---

We don't deprecate things in patches, and even if we did then we would indicate the patch version in the message only because it would make sense So to me the final `.0` is always implicit, we only need it when we deprecate in a patch version

If you look at all the other deprecate messages we almost never indicate the patch version number, although I did notice a few ones that did but to imho it's not really useful